src/specialitensornetworks.jl

```diff
@@ -0,0 +1,92 @@
+#RETURN THE IDENTITY TENSOR NETWORK (I.E. NETWORK WITH IDENTITY TENSORS ON EACH SITE) FOR A GIVEN GRAPH
+function identityITensorNetwork(g::NamedDimGraph; link_space = 2)
+    ψ = ITensorNetwork(g; link_space = link_space)
+  
+    for v in vertices(ψ)
+        is = inds(ψ[v])
+        ψ[v] = delta(is)
+    end
+    return ψ
+end
+
+#RETURN THE IDENTITY TENSOR NETWORK (I.E. NETWORK WITH IDENTITY TENSORS ON EACH SITE) FOR A GIVEN GRAPH
+function identityITensorNetwork(s; link_space = nothing)
+  ψ = ITensorNetwork(s; link_space)
+
+  for v in vertices(ψ)
+      is = inds(ψ[v])
+      ψ[v] = delta(is)
+  end
+  return ψ
+end
+  
+  #BUILD Z OF CLASSICAL ISING MODEL ON A GIVEN GRAPH AT INVERSE TEMP BETA
+  #H = -\sum_{(v,v') \in edges}\sigma^{z}_{v}\sigma^{z}_{v'}
+  #TAKE AS AN OPTIONAL ARGUMENT A LIST OF VERTICES OVER WHICH TO APPLY A SZ. THE RESULTANT NETWORK CAN THEN BE CONTRACTED AND DIVIDED BY THE ACTUAL PARTITION FUNCTION TO GET THAT OBSERVABLE
+  function ising_partition_function_ITensorNetwork(g::NamedDimGraph, beta::Float64; szverts=nothing)
```

Comment:
Maybe we can just call this `ising_network`?

Reply:
Good idea, I have renamed to ising_network and avoided code duplication by just building the indsnetwork and invoking the inds_network version of the function.